core/controllers/contributor_dashboard_test.py

```diff
@@ -615,38 +615,6 @@ def test_unpin_translation_opportunity(self) -> None:
             csrf_token=csrf_token,
             expected_status_int=200)
 
-    def test_raises_error_if_story_contain_none_exploration_id(self) -> None:
```

Comment:
Do we need a replacement test for this, or are we assuming that the None case is impossible?

Reply:
I was going to assume that the None case is impossible, since the mapping would be constructed without any `None` values inside the lists. Instead the exception should be raised when computing the topic summary. However, the logic for getting all linked exploration ids in a story occurs by calling the respective story's `StoryContents.get_all_linked_exp_ids()` method. I think inside that method would be the ideal place to raise an exception over a story node's missing exploration ID. Any thoughts on this?

Reply:
Yup that seems fine to me, just wanted to make sure we have test parity or that we're explicitly eliminating that case.